Poker.BE/Poker.BE.Domain/Game/Room.cs

```diff
@@ -92,9 +95,58 @@ public Room(Player creator, GamePreferences preferences) : this(creator)
         #endregion
 
         #region Methods
+        internal Player CreatePlayer()
+        {
+            var result = new Player();
+            activeAndPassivePlayers.Add(result);
+            return result;
+        }
+
+        /// <summary>
+        /// UC014 Start (Deal) a New Hand
+        /// </summary>
+        /// <see cref="https://docs.google.com/document/d/1OTee6BGDWK2usL53jdoeBOI-1Jh8wyNejbQ0ZroUhcA/edit#heading=h.3z6a7b6nlnjj"/>
         public void StartNewHand()
         {
-            CurrentHand = new Hand(deck, ActivePlayers);
+            if (ActivePlayers.Count < 2)
+            {
+                throw new NotEnoughPlayersException("Its should be at least 2 active players to start new hand!");
+            }
+            if (deck.Cards.Count != Deck.NCARDS)
+            {
+                throw new NotEnoughPlayersException("Cards must be dealt from a proper deck (standard 52-card deck containing no jokers)");
```

Comment:
sure, I used the only implemented exception (didnt ment to implement becuase conflict issues), will add todo for review the current exception and change to correct one when the exceptions impelemented